NumLib/Fem/CoordinatesMapping/NaturalCoordinatesMapping-impl.h

```diff
@@ -28,6 +32,7 @@ template <class T_MESH_ELEMENT, class T_SHAPE_FUNC, class T_SHAPE_MATRICES>
 inline void computeMappingMatrices(
         const T_MESH_ELEMENT &/*ele*/,
         const double* natural_pt,
+        const MeshLib::ElementCoordinatesMappingLocal &/*ele_local_coord*/,
```

Comment:
I think it's better to create the object outside of `computeMappingMatrices()`  because some `computeMappingMatrices()` internally call other `computeMappingMatrices()`, e.g. `computeMappingMatrices()` for `FieldType<ShapeMatrixType::DNDX>` calls `computeMappingMatrices()` for `FieldType<ShapeMatrixType::DNDR>`. `ElementCoordinatesMappingLocal` object is required in both functions.